backend/backend.go

```diff
@@ -12,7 +12,7 @@ import (
 
 const (
 	// DBString to format connection string to database for postgres
-	DBString = "host=%s port=%s dbname=%s user=%s password=%s sslmode=disable"
+	DBString = "host=%s port=%s dbname=%s user=%s password=%s sslmode=require"
```

Comment:
i will revert this line before create PR against upstream, it's needed for our tests

Reply:
I'm making some changes to make this configurable!

Reply:
We don't need this db change as the author already added them

Reply:
oh maaaaan!